### PR TITLE
fixing unit test

### PR DESCRIPTION
**Pull request by silverbp**
Just fixed a unit test, and starting to tweak the logging stuff..

### CODE REVIEW

File: src/logger.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // 3rd party
 var bunyan = require('bunyan');
 var PrettyStream = require('./pretty-stream');
-// Application
+var localStorage = require('continuation-local-storage');
```
Comment (jsonxr) [inline src/logger.js]: Why is this required? Doesn't look like the file is using it anywhere

Comment (silverbp) [inline src/logger.js]: I’m not.. I was going to add it in there to see if it works so it’s just laying around.. you can merge it as it is to prevent the amount of differences as I go through here if you want or you can wait.. it doesn’t really matter.. I’ll just push the branch back up and create another merge request.

Comment (silverbp) [inline src/logger.js]: I replied on email, but it doesn't look like it added my comment, I'm not using that yet.. I'm going to throw a quick line in there just to see if it works. If it doesn't you'll have some log entries without a correlating guid. But we don't have it at all right now, if it works, they'll be a correlating log id for all log entries..

Comment (silverbp) [inline src/logger.js]: But anyway, I was trying to get the unit test fix out of the way separately and forgot to remove that line.

Comment (jsonxr) [inline src/logger.js]: Can you go ahead and remove that line so our pull request stays focused?
